src/io/flutter/run/daemon/FlutterDaemonService.java

```diff
@@ -183,8 +183,8 @@ public FlutterApp startApp(@NotNull Project project,
                              @NotNull RunMode mode,
                              @Nullable String relativePath)
     throws ExecutionException {
-    boolean isPaused = mode.isDebug();
-    boolean isHot = mode.isDebug() ? HOT_MODE_DEFAULT : HOT_MODE_RELEASE;
+    boolean isPaused = mode.isReloadEnabled();
```

Comment:
Sorry, this pause is for the `--start-paused` flag, when starting apps. It should only be true for debugging configurations.